packages/r/rtl-sdr/package.yml

```diff
@@ -18,5 +18,3 @@ build      : |
 install    : |
     %make_install
     install -D -m00644 rtl-sdr.rules $installdir/%libdir%/udev/rules.d/10-rtl-sdr.rules
-
-    rm $installdir/usr/lib/*.a
```

Comment:
What was the reason to remove this line?

Reply:
It does not exist in the updated version:

+ rm '/home/build/YPKG/root/rtl-sdr/install/usr/lib/*.a'
rm: cannot remove '/home/build/YPKG/root/rtl-sdr/install/usr/lib/*.a': No such file or directory

Reply:
Ok, I understand what happened. Static lib was added actually to `/usr/lib64/librtlsdr.a`
Use this instead `rm  $installdir/%libdir%/*.a`